fla/layers/attn.py

```diff
@@ -159,7 +159,7 @@ def forward(
                 causal=True,
                 window_size=(-1, -1) if self.window_size is None else (self.window_size-1, 0)
             )
-        o = o.reshape(batch_size, q_len, self.hidden_size)
+        o = o.reshape(batch_size, q_len, -1)
```

Comment:
The hidden_size can change when tp is enabled, but we can leave it inferred by torch